suse/rdma-core.spec

```diff
@@ -342,6 +348,10 @@ easy, object-oriented access to IB verbs.
 
 %prep
 %setup -q -n  %{name}-%{version}%{git_ver}
+%if %{with_prebuilt_doc}
+#Extract prebuilt pandoc file in the buildlib directory
```

Comment:
We prefer to keep specs as close as possible to shipped version.

Why doesn't cbuild fit for you?

Reply:
The reason for all these weird macros (curlmini too) is that rdma-core is part of the "core" package of the distro that everything is being built on.
(For some reason boost requires openmpi which pulls rdma-core).
Therefore any dependency we add to the rdma-core.spec file on SUSE needs to be added to these "core" packages during validation. And stuff like pandoc should really not end up in there.

Looking into our build system, It seems like someone might have fixed that. So this patch may not be required after all.

Reply:
Scratch that last part. We managed to drop one of the dependencies but there's no way we're getting rdma-core out of this "core" packages.

Reply:
Even so, the way to avoid pandoc is to include the prebuilt files into the main .tar.gz - so why add another mechanism?? Do you need a cbuild script to build that ..tar.gz or something?

Reply:
Because our build system needs to be able to consistently redownload the source tarball (meaning running git archive and get the same result).
Which means all additional files needs to be outside of the tar.gz

You can see what it looks like here https://build.opensuse.org/package/show/science:HPC/rdma-core

Reply:
It is really weird that you can use pandoc to make a tar file but not run it as part of the normal build. But if that is what you want to do.. then sure

Reply:
Because I do this offline and commit the generated docs into our VCS.
The nit in the spec just extract it during the real build (to mimic a release tarball) and generates the RPMs

Reply:
I removed these changes. I agree this is not pretty and will hack something on SUSE build system side